index.html

```diff
@@ -45,5 +45,149 @@ <h1>I’m Roodjino Glad to see you!</h1>
+                    <a href="#"> <button type="button">See Project</button> </a>
+                </div>
+                
+            </div>
+
+            
+            <div class="card-work">
+                <div class="card-img">
+                    <img src="images/portfolio4.png" alt="snapshoot">
+                </div>
+                    
+                <div class="card-text">
+                    <h2>Project Title</h2>
+                    <ul class="project-details">
+                        <li>Canopy</li>
+                        <li>Black end Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="project-stack">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <a href="#"> <button type="button">See Project</button> </a>
+                </div>
+                
+            </div>
+
+        </section>
+
+
+        <section class="about">
+
+            <div class="container">
+                <div class="text">
+                    <h1>I’m Roodjino Glad to see you!</h1>
+                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                </div>
+                <div class="connect">
+                    <p> LET'S CONNECT</p>
+                    <ul class="logo"> 
+                        <li><a href="#"><img src="images/twitter.svg"> </a> </li>
+                        <li><a href="#"><img src="images/linkedin.svg"> </a> </li>
+                        <li><a href="#"><img src="images/medium.svg"> </a> </li>
+                        <li> <a href="#"><img src="images/github.svg"> </a> </li>
+                        <li> <a href="#"><img src="images/peace.svg"> </a></li>
+                    </ul>
+                  <a href="#"><button type="button">Get my resume</button></a>
+                </div>
+            </div>
+            <div class="about-stack">
+
+               <span class="lan"> <p>Languages</p> <img src="images/hov-dow.png"/> </span>
+
+                <ul class="languages">
+                    <li class="lan-items"><span> <img src="images/js.svg"/></span>JavaScript</li>
+                    <li class="lan-items"><span> <img src="images/html.svg"/></span>Html</li>
+                    <li class="lan-items"><span> <img src="images/css.svg"/></span>Css</li>
+                </ul>
+
+                <span class="lan"> <p>Framework</p> <img src="images/hov-down.png"/> </span>
+
+                <span class="lan">  <p>Skills</p> <img src="images/hov-down.png"/> </span>
+    
+
+            </div>
+
+        
+
+        </section>
+
+
+
     </body>
```

Comment:
[OPTIONAL] please try as much as possible to organise and format your code without leaving unnecessary empty spaces since it can make your code bulky thereby reducing performances